apollo-client/src/test/java/com/ctrip/framework/apollo/spring/JavaConfigAnnotationTest.java

```diff
@@ -281,10 +366,286 @@ public void testApolloConfigChangeListenerWithYamlFile() throws Exception {
     assertEquals(anotherValue, yamlConfig.getProperty(someKey, null));
   }
 
-  private <T> T getBean(Class<T> beanClass, Class<?>... annotatedClasses) {
-    AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(annotatedClasses);
+  @Test
+  public void testApolloConfigChangeListenerResolveExpressionSimple() {
+    // for ignore, no listener use it
+    Config ignoreConfig = mock(Config.class);
+    mockConfig("ignore.for.listener", ignoreConfig);
+
+    Config applicationConfig = mock(Config.class);
+    mockConfig(ConfigConsts.NAMESPACE_APPLICATION, applicationConfig);
+
+    getSimpleBean(TestApolloConfigChangeListenerResolveExpressionSimpleConfiguration.class);
+
+    // no using
+    verify(ignoreConfig, never()).addChangeListener(any(ConfigChangeListener.class));
+
+    // strange, it invoke 2 times
```

Comment:
because there is an auto update feature enabled: `com.ctrip.framework.apollo.spring.config.PropertySourcesProcessor#initializeAutoUpdatePropertiesFeature`